### DISCUSSION TITLE
Create new shapes with <include-shape>

### DISCUSSION THREAD

**Discussion by LubaaBalakhashvili** [jgraph/drawio-desktop#1439] [Q&A]
I want to create shapes with same base and if I change that base, new shapes to be changed too. In documentation I read about<include-shape> but I can't understand how to use this and can't find anything else about it. How can I use this sub-shapes?

Link of documentation: https://www.drawio.com/doc/faq/shape-complex-create-edit
![image](https://github.com/jgraph/drawio-desktop/assets/102235002/0d24b85f-ae66-426e-936c-fe0d09c66b6a)

**Comment by TechGeek01** [comment]
I've been searching for this answer for at least a year. I would also like to know how to do this. The `shape` attribute in the style for a built-in shape works, but not for shapes that are custom.

**Reply by aleksandarsusnjar** [reply]
I'm looking for this too. Nothing to be found anywhere. Searched through the source to find some examples ... nothing. Asked the same question elsewhere: https://github.com/jgraph/drawio/discussions/4517

**Comment by aleksandarsusnjar** [comment]
Created an issue from this: https://github.com/jgraph/drawio-desktop/issues/1806
